src/commandProcessExecution.ts

```diff
@@ -32,6 +32,11 @@ export class CommandProcessExecutionFactory {
   }
 }
 
+export interface CommandProcessResult {
+  stdout: string;
```

Comment:
Please also add a field that contains the full output in the order that it is received.